features/support/run_helpers.rb

```diff
@@ -91,7 +91,9 @@ def run_shell_command command, inputs = []
 
 
 def shell_overrides
-  "PATH=#{SOURCE_DIRECTORY}:#{SHELL_OVERRIDE_DIRECTORY}:$PATH; export WHICH_SOURCE=#{TOOLS_INSTALLED_FILENAME}"
+  "PATH=#{SOURCE_DIRECTORY}:#{SHELL_OVERRIDE_DIRECTORY}:$PATH;"\
+  "export WHICH_SOURCE=#{TOOLS_INSTALLED_FILENAME};"\
+  "export GT_ENV=test"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.